ui/app/components/transit-key-actions.js

```diff
@@ -4,7 +4,6 @@
  */
 
 import { assign } from '@ember/polyfills';
-import { copy } from 'ember-copy';
```

Comment:
`ember-copy` is pretty old, and there are better, more modern ways to copy objects. Shallow objects can be copied with ES6 object spread, and deeply nested objects can use the native [`structuredClone`](https://developer.mozilla.org/en-US/docs/Web/API/structuredClone) function. Those two are what I used to remove `ember-copy` usage.